src/source/feeds.rst

```diff
@@ -31,6 +78,635 @@ XML file.
+
+Allowed image formats: JPEG, JPG, PNG, GIF, BMP. 
+
+All images will be resized if necessary to a size of maximum 1024px height and 1024px width (preserving the aspect ratio)
+The system will download the images and, if they meet the requirements, store them on our servers in several sizes.
+
+The images will be presented in the order as provided. The first image is shown in search results and as the main image on the item page.
+
+======= =========================================================================================================
+Example .. code-block:: html 
+
+            <ad:media>
+                <ad:image url="https://images.pexels.com/photos/62289/yemen-chameleon-chamaeleo-calyptratus-chameleon-reptile-62289.jpeg"/>
+                <ad:image url="https://images.pexels.com/photos/47547/squirrel-animal-cute-rodents-47547.jpeg?auto=compress&amp;cs=tinysrgb&amp;w=1260&amp;h=750&amp;dpr=2"/>
+            <ad:media/>
+======= =========================================================================================================
+
+.. index:: attributes
+.. _feed_attr:
+
+attributes
+""""""""""
+
+Optional collection of ad :ref:`user_defined_attributes` (category-dependent). 
+Note that categories can have mandatory attributes, for which a default value will be filled in if not supplied.
+
+======= =========================================================================================================
+Example .. code-block:: html 
+
+            <ad:attributes>
+                <ad:attribute>
+                    <ad:attributeName>color</ad:attributeName>
+                    <ad:attributeLocale>nl</ad:attributeLocale>
+                    <ad:attributeLabel>Kleu</ad:attributeLabel>
+                    <ad:attributeValue>Rood</ad:attributeValue>
+                </ad:attribute>
+                <ad:attribute>
+                    <ad:attributeName>color</ad:attributeName>
+                    <ad:attributeLocale>en</ad:attributeLocale>
+                    <ad:attributeLabel>Color</ad:attributeLabel>
+                    <ad:attributeValue>Red</ad:attributeValue>
+                </ad:attribute>
+                <ad:attribute>
+                    <ad:attributeName>Model</ad:attributeName>
+                    <ad:attributeValue>Slim</ad:attributeValue>
+                    <ad:attributeValue>Spro</ad:attributeValue>
+                </ad:attribute>
+            </ad:attributes>
+
+======= =========================================================================================================
+
+.. index:: budgetDetails
+.. _feed_budget:
+
+budgetDetails
+"""""""""""""
+
+Section with budget details
+
+============= ========================================== ========
+Name          Description                                Required
+============= ========================================== ========
+autobid       use auto bidding option true/false         No
+cpc           CPC for the given ad in cents              Yes
```

Comment:
is CPC mandatory in feeds? I was under the impression that if not provided, we set to the minimum of the category. Unless you're changing this, then all OK

Reply:
seems like it has been nillable indeed (but with the combo of `minOccurs="1"` I am not sure what happens tbh
```
            <xs:element name="cpc" type="xs:string" nillable="true" minOccurs="1" maxOccurs="1"/>
```

Reply:
The entire budget section is optional, but if one decides to provide it then it must come with CPC according to xsd.

Reply:
yeah, but the element is nillable; this afaik means the element must be there but can have empty value, like `<cpc />` or `<cpc><cpc>`

Reply:
ah!

Reply:
does anyone do this, I wonder